### PR TITLE
Update READMEs.

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,3 +210,7 @@ ninja report-coverage
 
 Then the coverage report can be found under the `$BUILD_DIR/coverage-report`
 directory.
+
+[khr-glslang]: https://github.com/KhronosGroup/glslang
+[google-glslang]: https://github.com/KhronosGroup/glslang
```
Comment (dneto0) [inline README.md]: Should be google/glslang.

File: README.md [hunk 2]
```diff
@@ -50,18 +49,17 @@ for more information. See also the [`AUTHORS`](AUTHORS) and
...
+before it appears upstream in [KhronosGroup/glslang][khr-glslang]
 We intend to upstream all changes to glslang. We maintain the separate
 copy only to stage those changes for review, and to provide something for
 Shaderc to build against in the meantime.  Please see
 [DEVELOPMENT.howto.md](DEVELOPMENT.howto.md) for more details.
 
-Shaderc depends on [SPIRV-Tools](https://github.com/KhronosGroup/SPIRV-Tools)
-for assembling and disassembling SPIR-V binaries.
+Shaderc depends on [SPIRV-Tools][spirv-tools] for assembling and disassembling
+SPIR-V binaries.
```
Comment (dneto0) [inline README.md]: "assembling, disassembling, and transforming SPIR-V binaries."

(we use it to strip debug info, for example).

File: README.md (hunk 1 above)
Comment (antiagainst) [inline README.md]: Good catch! Done.

File: README.md (hunk 2 above)
Comment (antiagainst) [inline README.md]: Done.

**Pull request merged**